2_adventures/001/kotp/04_automated_testing_example/spec/fizz_buzz_spec.rb

```diff
@@ -0,0 +1,53 @@
+require_relative 'spec_helper'
+require 'fizz_buzz'
+
+describe FizzBuzz do
+  describe 'Single Numbers' do
+    it 'must return 1 for 1' do
```

Comment:
Reading the 'story' though, I would see "returns fizz" and wonder "What returns fizz?  Or Why?"  So for me having the input listed tells me a lot about the functionality, rather than having to go into the code.  Another thing I did not do in this spec is give it a message in case it fails...